vendor.mod

```diff
@@ -256,3 +256,5 @@ exclude (
 	github.com/davecgh/go-spew v1.1.2-0.20180830191138-d8f796af33cc
 	github.com/pmezard/go-difflib v1.0.1-0.20181226105442-5d4384ee4fb2
 )
+
+replace github.com/moby/sys/user v0.3.0 => github.com/moby/sys/user v0.0.0-20250227193707-71f0c5ead442
```

Comment:
@dmcgowan I tagged v0.4.0, so you can remove this replace rule

I was about to open a PR to update the version, but we probably want to do that as part of this PR so that it's clear that files were moved from here to the other module.